fractions/microprofile/microprofile-restclient/src/main/java/org/wildfly/swarm/microprofile/restclient/RestClientFraction.java

```diff
@@ -15,17 +15,16 @@
  */
 package org.wildfly.swarm.microprofile.restclient;
 
+import static org.wildfly.swarm.spi.api.annotations.DeploymentModule.MetaInfDisposition.IMPORT;
```

Comment:
Is this still needed ? My basic understanding is that it is only needed if some work is being done in the 'deployment' package